.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.0
```

Comment:
Sorry I didn't mean for it to get there. I couldn't compile Jekyll without adding it but I could have added it to the gitignore. (still can). Do you want me to remove?